### PR TITLE
S03_US4 - DB_Loop

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved!

File: services.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+--
+-- Base de données :  `simple-mvc`
+--
+
+-- --------------------------------------------------------
+--
+-- Structure de la table `services`
+--
+
+CREATE TABLE IF NOT EXISTS `services` (
```
Comment (LucyMerlier) [inline services.sql:28]: good practice is for table names to be singular

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
good practice is to use singular ("Services" => "Service")

**Pull request closed without merging**